### PR TITLE
Stele Road

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -1905,24 +1905,24 @@ QUEST_LV_0100_20150317_001900	{memo X}$We are the believers of Goddess Saule.{nl
...
 QUEST_LV_0100_20150317_001915	$I will pray for the Goddess to bless you.{nl}You should stop him no matter what.
-QUEST_LV_0100_20150317_001916	$I came here after hearing that her epitaph is here, but it will be difficult to find it all alone.{nl}I hope you can be my ally. Are you interested?
-QUEST_LV_0100_20150317_001917	$He is the traitor of the Kadumel's era and the owner of Tower of the Earth.{nl}He was regarded as the best swordsman even when he was young and many people followed him.
-QUEST_LV_0100_20150317_001918	$His was too greedy, so he started a coup de tat.{nl}If the great merchant, Premier Eminent wasn't there, then history could have been much different.
-QUEST_LV_0100_20150317_001919	$Ruklys and I were taught by the same teacher.{nl}That's why I know what he is trying to achieve.
+QUEST_LV_0100_20150317_001916	$I came here after hearing that her epitaph is here, but it will be difficult to find it all alone.{nl}I hope you can be an ally. Are you interested?
+QUEST_LV_0100_20150317_001917	$He is the traitor of the Kadumel's era and the owner of Tower of the Earth.{nl}He was regarded as the best swordsman, even when he was young, and many people followed him.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Kadumel's era - > Kadumel Era
Tower of the Earth -> Earth Tower

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -7665,8 +7665,8 @@ QUEST_LV_0200_20150714_007668	This farm's owner is Gina Green.{nl}They kindly le
 QUEST_LV_0200_20150714_007669	Hey you. Revelator. Don't just look at me. Help me.{nl}Please kick out Lizard men on my fields.
 QUEST_LV_0200_20150714_007670	You must know that there isn't enough food in the kingdom.{nl}If you are okay, can you help me with my investigation?
 QUEST_LV_0200_20150714_007671	That's really appreciated.[nl}Firstly, collect the lumps of hair from Rambear Archers. I have something to check.
-QUEST_LV_0200_20150714_007672	If that's the case, please eliminate all short trees.{nl}Ah, don't forget to bring the evidences as well. I need something to report.
-QUEST_LV_0200_20150714_007673	Even if you are the Revelator, this is a private land. {nl}Go out if you don't have any business here!{nl}
+QUEST_LV_0200_20150714_007672	If that's the case, please eliminate all the Stumpy Trees.{nl}Ah, don't forget to bring evidence as well. I need something to report.
+QUEST_LV_0200_20150714_007673	Even if you are the revelator, this is a private land. {nl}Go out if you don't have any business here!{nl}
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:7669]: Why the lowercase?

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**